axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -187,6 +193,19 @@ def test_all_states_reachable(self):
         extra_info = 'The following states are un-reachable: {}'.format(un_callable_states)
         self.assertEqual(un_callable_states, set(), msg=extra_info)
 
+    def test_strategy(self):
+        """A particular FSM test"""
+        transitions = [(0, C, 0, C), (0, D, 3, C), (1, C, 5, D),
```

Comment:
very very minor.  all the transitions are tuple[tuple].  mypy gave me no end of trouble when they were either list or tuple.